src/main/java/com/box/sdk/SharedLinkAPIConnection.java

```diff
@@ -14,7 +14,7 @@
     }
 
     SharedLinkAPIConnection(BoxAPIConnection connection, String sharedLink, String sharedLinkPassword) {
-        super(null);
+        super("");
```

Comment:
super(null) can lead to two Constructors BoxAPIConnection(String accessToken) or BoxAPIConnection(BoxConfig boxConfig)